GameData/NearFutureSpacecraft/Localization/ru.cfg

```diff
@@ -0,0 +1,150 @@
+    #LOC_NFSpacecraft_monoprop-tank-125-1_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-125-2_title = FL-R-A375 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-125-2_description = «Рокомакс» предлагает этот набор топливных баков в изящном форм-факторе 1.25м! Это бак средний, в са-амую тютельку.
+    #LOC_NFSpacecraft_monoprop-tank-125-2_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-125-3_title = FL-R-A185 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-125-3_description = «Рокомакс» предлагает этот набор топливных баков в изящном форм-факторе 1.25м! Этот бак малый.
+    #LOC_NFSpacecraft_monoprop-tank-125-3_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-25-1_title = FL-R-B3000 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-25-1_description = Серия баков FL-R хранит монотопливо в весьма изобильном количестве.
+    #LOC_NFSpacecraft_monoprop-tank-25-1_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-25-2_title = FL-R-B1500 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-25-2_description = Серия баков FL-R хранит монотопливо в весьма изобильном количестве, хотя, в этой полуразмерной модели, чуток поменьше.
+    #LOC_NFSpacecraft_monoprop-tank-25-2_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-25-3_title = FL-R-B750 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-25-3_description = Допустим, вы хотите, чтобы ваш 750-литровый бак монотоплива выглядел симпатично. Этот бак может быть лучшим выбором, нежели FL-R-1.
+    #LOC_NFSpacecraft_monoprop-tank-25-3_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-375-1_title = FL-R-C6000 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-375-1_description = Говорят, что вам на самом деле не нужно столько монотоплива для ваших РСУ в космосе. Не слушайте их.
+    #LOC_NFSpacecraft_monoprop-tank-375-1_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-375-2_title = FL-R-C3000 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-375-2_description = Бак больше-среднего-размера вмещает топлива больше-среднего-объёма. В данном случае - монотоплива.
+    #LOC_NFSpacecraft_monoprop-tank-375-2_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-375-3_title = FL-R-C1500 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-375-3_description = Строите большой технический отсек? Вам нужен большой монотопливный бак? Это то, что вам нужно.
+    #LOC_NFSpacecraft_monoprop-tank-375-3_tags = fueltank mono monopropellant propellant nearfuture
+
+    // Structural
+    #LOC_NFSpacecraft_engine-pod-1_title = SD-01 Гондола двигателя радиальной установки		// Why do you use aerocraft terms for spacecraft parts? 
```

Comment:
Alright, you're free to disagree then... change the russian one to whatever you like.